docs/wiki.md

```diff
@@ -1,59 +1,66 @@
+# Wiki
+
 ## Overview
-We’ve rebuilt the most popular 100+ FxCop rules as live analyzers to help you detect problems in your code and make it super easy for you to fix them on the spot with quick fixes. Installing the NuGet or Extension gives you the same great code analysis as before with FXCop but now the analysis is live as-you-type and has quick-fixes.
 
-## Install Instructions
-You can download the FXCop live code analyzers as a [NuGet package](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/) or as a Visual Studio extension ([VS2017](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.MicrosoftCodeAnalysis2017) or [VS2019](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.MicrosoftCodeAnalysis2019)).
+We've rebuilt the most popular FxCop rules and additional more rules as live analyzers to help you detect problems in your code and make it super easy for you to fix them on the spot with quick fixes. You can get the analyzers via [`Microsoft.CodeAnalysis.NetAnalyzers` NuGet package](https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers) or from the .NET SDK. This gives you the same great code analysis as before with FXCop but now the analysis is live as-you-type and has quick-fixes.
```

Comment:
I think we should avoid mentioning specific NuGet package, or .NET SDK or any acquisition modes here and instead just add a help link to https://learn.microsoft.com/dotnet/fundamentals/code-analysis/overview?tabs=net-7 which has all the relevant information.